### PR TITLE
Adds te: trailers header

### PR DESCRIPTION
**Pull request by ignasi35**
References #1173

### CODE REVIEW

File: runtime/src/main/scala/akka/grpc/internal/GrpcRequestHelpers.scala [hunk 1]
```diff
@@ -6,15 +6,15 @@ package akka.grpc.internal
 
 import akka.actor.ActorSystem
 import akka.actor.ClassicActorSystemProvider
-import akka.grpc.{ ProtobufSerializer, Trailers }
+import akka.grpc.{ProtobufSerializer, Trailers}
```
Comment (raboof) [inline runtime/src/main/scala/akka/grpc/internal/GrpcRequestHelpers.scala:9]: Might need a format?

**Review by raboof — APPROVED** [review]

**Comment by ignasi35** [comment]
The failure in https://travis-ci.com/github/akka/akka-grpc/builds/200051980 seems flaky. The same tests passed in other jobs of the same build and the build passed on the `Travis CI- PR` check.

**Comment by ignasi35** [comment]
> The failure in [travis-ci.com/github/akka/akka-grpc/builds/200051980](https://travis-ci.com/github/akka/akka-grpc/builds/200051980) seems flaky. The same tests passed in other jobs of the same build and the build passed on the `Travis CI- PR` check.

Hmm, the test is not flaky. It just fails on JDK11 + scala 2.13.

**Comment by raboof** [comment]
> The failure in https://travis-ci.com/github/akka/akka-grpc/builds/200051980 seems flaky. The same tests passed in other jobs of the same build and the build passed on the `Travis CI- PR` check.

I'm pretty sure that's the flakey test failure that #1178 is intended to fix. Rebuilding now.

**Comment by raboof** [comment]
(failure was #935, restarted)

**Pull request merged**